compiler/code-gen/files/init-scripts.cpp

```diff
@@ -65,6 +65,14 @@ void StaticInit::compile(CodeGenerator &W) const {
     W << END << NL << NL;
   }
 
+  if (G->is_output_mode_k2()) {
+    W << "namespace kphp::compiler_interface {" << NL;
+    FunctionSignatureGenerator(W) << ("std::string_view get_main_file_name() ") << BEGIN;
```

Comment:
let's make this functions `constexpr`

Reply:
`clang++` gives error with message that `inline function ... is not defined`